ballerina-to-openapi/src/main/java/io/ballerina/openapi/service/mapper/ServersMapperImpl.java

```diff
@@ -266,4 +271,22 @@ private static String concatenateServerURL(String host, SeparatedNodeList<Mappin
         }
         return host;
     }
+
+    private static Server getDefaultServerWithBasePath(String serviceBasePath) {
+        String serverUrl = String.format("http://{server}:{port}%s", serviceBasePath);
```

Comment:
Does this template change according to the port? ex: if the port is 443, we ideally map it to https:

Reply:
This is added as a dummy server schema for service contract type since it is not possible to get the listener details. When we implement a service with this contract, we will replace these default values from the listener.